### DISCUSSION TITLE
Ban lists seem to be resetting on every large content update

### DISCUSSION THREAD

**Discussion by Holneer** [FakeFishGames/Barotrauma#14976] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

For some reason whenever I come back to play after a new big content drop my ban lists just get purged. This has happened 3 times to me already. I believe only once did it not reset after a big update. This is big issue to me as I mostly do public campaigns and having to suffer all the griefers again (Probably had like 100+ people banned across all 3 lists) is annoying at least.

One person from the Official Barotrauma discord server theorized what could be the potential cause:
"might be default baro comes with server setting files and overwrites yours lol"

### Reproduction steps

1. Play the game, ban people.
2. Wait for a big new update.
3. Play again, but now the ban lists are clean.

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

The ban list ("Data/bannedplayers.xml") is not included in the game install, but generated by the game, so updating the game should normally never clear it. I wonder if you've uninstalled and reinstalled the game in between the updates, or perhaps "verified integrity of game files" which essentially wipes the game's install folder into a clean slate? In any case, we need to look into this - enabling Steam's cloud syncing on this file could for example make it less likely for the lists to get accidentally lost.

**Reply by Holneer** [reply]
I do have cloud turned on, didn't verify my game nor delete/reinstall it, didn't even move it from drive to drive. The closest I've had to that was updating my workshop sub or participating in an Unstable beta, but I believe my first ever ban list reset happened prior to me even knowing there are beta branches. And since ban list is in the game's files rather than AppData I doubt me reinstalling Windows had anything to do with it, at least it wouldn't justify the other two resets.

The only things that seem to reset in all of these 3-4 years of me playing are my local submarine files (for which I had to download a local copy from the workshop) and the ban lists. Saves/campaign still have some files dating back to 2020-2021.
